DOCS-600: plant watering tutorial and code updates (#963)
viamrobotics/docs: docs/tutorials/projects/make-a-plant-watering-robot.md

> 

Reviewer: Can't comment on line 63 but "Analog to Digital Converter" should become "analog-to-digital converter," and I think "small oval" -> "half circle"

Next, wire your capacitive soil moisture sensor to your Pi and ADC.

> Refer to the following pinout diagram for the blue module part of your capacitive soil moisture sensor:

Reviewer: ```suggestion
Here is a detailed view of the capacitative soil moisture sensor module:
```

![Pinout diagram for the capacitive soil moisture sensor.](../../img/plant-watering-pi/moisture-sensor-pinout.png)

> Start by connecting the female jumper wires at the end of the sensor prongs to the blue module where the diagram shown above is labeled "Connect with Probe."

Reviewer: ```suggestion
Note the pins on the left side of the diagram above labeled "Connect with Probe."
Connect the sensor probe prongs to these pins with female jumper wires.
```

Then, add your pump as a [motor component](/components/motor/) by adding a new component with **type** `motor` and **model** `gpio`.

> Add your board (in this example, named `local`) to this component's dependencies (`depends_on`) and set the attributes **Max RPM** to `1000` and **PWM**  to `12 GPIO 18` (the number and GPIO number of the pin that you wired the pump's PWM to).

Reviewer: ```suggestion
Select your board (in this example, named `local`) from the **Board** drop-down and set the attributes **Max RPM** to `1000` and **PWM**  to `12 GPIO 18` (the number and GPIO number of the pin that you wired the pump's PWM to).
```
You do not need to put it in the depends_on because that's taken care of implicitly when you add it as the board. I recommend taking it out of the example JSON as well. If you choose to leave it in, it should at least be indented in the JSON.

Add your sensor server as a [remote part](/manage/parts-and-remotes/) called `my-sensor-server`:

{{< tabs >}}
{{% tab name="JSON Template" %}}

```json {class="line-numbers linkable-line-numbers"}
// "components": [ board & motor ... ],
"remotes": [
    {
      "name": "my-sensor-server",
      "insecure": true,
      "address": "localhost:9090"
    }
]
```

{{% /tab %}}
{{% tab name="Config Builder" %}}

![Creation of a custom sensor remote in the Viam app config builder."](../../img/plant-watering-pi/sensor-remote-config-builder.png)

{{% /tab %}}
{{< /tabs >}}

{{% alert title="Caution" color="caution" %}}

> If you use the Config Builder to make your remote, make sure you have still added the line: `"insecure": true,` to the Raw JSON.

Reviewer: I would eliminate all the "select" mode stuff and point them to JSON mode (within config builder) because it's not really equivalent and may just cause confusion? It seems that the result of the "config builder" screen shot here would be a very different bunch of raw JSON than is shown on the other tab.
Additionally, the link to remote configuration instructions doesn't include "SELECT" config stuff right now because it currently is actually easier to just copy paste the whole thing into "JSON" mode instead of using "Select" mode. So, users won't have instructions for how to do select mode correctly, currently.

Now, you can add code into <file>plant-watering-robot.py</file> to write the logic that defines your plant watering system.

> To start, you can your system logic code into the `main()` function of the program.

Reviewer: ```suggestion
To start, add your system logic code into the `main()` function of the program.
```
I think this is what is meant here?
